cypress/e2e/applicant/apply.cy.js

```diff
@@ -0,0 +1,226 @@
+require("dotenv").config();
+
+const BASE_URL =
+  "https://dev-env.find-a-grant-support-dev.service.cabinetoffice.gov.uk/";
+
+const signInWithOneLogin = () => {
+  cy.contains("Sign in with GOV.UK One Login").click();
+
+  cy.origin("https://signin.integration.account.gov.uk", () => {
+    cy.contains("Sign in").click();
+    cy.get('[name="email"]').type(Cypress.env("oneLoginApplicantEmail"));
+    cy.contains("Continue").click();
+    cy.get('[name="password"]').type(Cypress.env("oneLoginApplicantPassword"));
+    cy.contains("Continue").click();
+    //cy.contains("Continue").click();
```

Comment:
rm

Reply:
this button click is actually necessary if you need to accept OL T&Cs, need to identify when that occurs so I can do it programmatically so it's kind of a reminder that this is going to need done

Reply:
Added a comment above this line with a reference to the ticket to solve the issue